interface/src/app/node/node.js

```diff
@@ -92,5 +92,204 @@ angular.module('adage.node', [
   }]
 )
 
+.directive('highRangeExp', ['$http', '$log', function($http, $log) {
+  return {
+    templateUrl: 'node/high_range_exp.tpl.html',
+    restrict: 'E',
+    scope: {
+      nodeId: '@'
+    },
+    link: function($scope) {
+      $scope.queryStatus = 'Connecting to the server ...';
+      $scope.activities = {};
+      $scope.experiments = [];
+      $scope.top20Mode = true;
```

Comment:
Can this number be passed in as a parameter to tag `<high-range-exp node-id="{{ctrl.id}}">`? I am imagining that we might want to re-use this in other places with potentially fewer or more experiments shown by default.